sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/CommunicationIdentityAsyncClient.java

```diff
@@ -228,56 +267,102 @@ public Mono<Response<Void>> revokeTokensWithResponse(CommunicationUserIdentifier
     }
 
     /**
-     * Gets a token for an identity.
+     * Gets a Communication Identity access token for a {@link CommunicationUserIdentifier}.
      *
-     * @param communicationUser The user to be issued tokens.
-     * @param scopes The scopes that the token should have.
-     * @return The access token.
+     * @param communicationUser A {@link CommunicationUserIdentifier} from whom to issue a Communication Identity
+     * access token.
+     * @param scopes List of {@link CommunicationTokenScope} scopes for the Communication Identity access token.
+     * @param tokenExpiresAfter Custom validity period of the Communication Identity access token within [1,24]
+     * hours range. If not provided, the default value of 24 hours will be used.
+     * @return the Communication Identity access token.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<AccessToken> getToken(CommunicationUserIdentifier communicationUser,
-        Iterable<CommunicationTokenScope> scopes) {
+        Iterable<CommunicationTokenScope> scopes, Duration tokenExpiresAfter) {
         try {
-            Objects.requireNonNull(communicationUser);
```

Comment:
I guess you should revert it, the communicationUser should be checked first for nullability

Reply:
also we can specify  an error message as well `Objects.requireNonNull(communicationUser, "'communicationUser' cannot be null.");` same for other occurrences

Reply:
Order reverted.